tests/general/cloc_test_files/example2.py

```diff
@@ -0,0 +1,7 @@
+# This comments are for testing purpose
+# This program will just try to add two static numbers
+
+first = 5  # First number
+second = 10  # Second number
+
+sum = first + second
```

Comment:
@meetmangukiya @SanketDG 
Although this commit is outdated, I believe that suggested patch by @gitmate-bot  is not appropriate. It should just add one line at the end instead of removing the last line just because 2nd last line is empty.
May be something is wrong with configs for @gitmate-bot  checking.